src/routes/blog/post/scan-receipts-with-appwrite-functions/+page.markdoc

```diff
@@ -122,12 +122,9 @@ import os
 def main(context):
     # Why not try the Appwrite SDK?
     #
-    # client = (
-    #     Client()
-    #     .set_endpoint("https://cloud.appwrite.io/v1")
-    #     .set_project(os.environ["APPWRITE_FUNCTION_PROJECT_ID"])
-    #     .set_key(os.environ["APPWRITE_API_KEY"])
-    # )
+    # client = Client()
+    # // Only need to set key, the rest "just works"
```

Comment:
> the rest "just works"

I don't think this matches the tone/voice we want for docs/tutorials. How can this be rephrased?